### PR TITLE
adds overflow ellipsis to pipeline metadata

### PR DESCRIPTION
**Pull request by astewes**
*  Adds ellipsis to overflowing metadata name + description
*  Removes some styling for metadata inputs that is no longer necessary

* Ellipsis confirmed to already be working for pipeline detail view

description

### CODE REVIEW

File: cdap-ui/app/features/hydrator/toppanel.less [hunk 1]
```diff
@@ -58,34 +59,56 @@ body.theme-cdap {
...
+                &:before {
+                  content: "[Pipeline Name]";
+                }
+              }
+            }
+            &:last-child {
+              font-size: 12px;
+              &.placeholder {
+                &:before {
+                  content: "Enter a description for your pipeline";
```
Comment (ajainarayanan) [inline cdap-ui/app/features/hydrator/toppanel.less]: Could be a minor thing but if we have ```[``` & ```]``` around Pipeline Name placeholder should we have one around this too?

**Comment by ajainarayanan** [comment]
I have seen this quite a number of times. Right now we don't have a link to go to Hydrator home view if we click on "Cask Hydrator" text on the top left of the create studio view. However if you click on the edges of "Name & Description" box in create studio view we go back. This might be a bad because the user might be editing the draft and at the end if they click on the edges to rename the draft or to add description we go back to list view without saving the draft.

We might want to fix this issue in this PR (or in another one. I would suggest another one) since we are dealing with fixing Name & Description in Studio view.

goingback

File: cdap-ui/app/features/hydrator/toppanel.less (hunk 1 above)
Comment (astewes) [inline cdap-ui/app/features/hydrator/toppanel.less]: Per Brady, [Pipeline Name] has been changed to "Name your pipeline"

**Comment by astewes** [comment]
@ajainarayanan let's fix the linking issue in a follow-up PR. I will make note of it.

**Comment by ajainarayanan** [comment]
:+1: LGTM. Myself along with @sreevatsanraman reviewed.

**Pull request merged**